internal/drand-cli/cli.go

```diff
@@ -344,15 +344,15 @@ var appCommands = []*cli.Command{
 	},
 	{
 		Name:  "share",
-		Usage: "Launch a sharing protocol.",
+		Usage: "The old command for running DKGs; this has been removed",
 		Action: func(c *cli.Context) error {
 			banner(c.App.Writer)
 			return deprecatedShareCommand(c)
 		},
 	},
```

Comment:
Why do we keep it? And why don't we "hide" it then?

Reply:
didn't know it was possible - have added it!